### DISCUSSION TITLE
PCM5102A

### DISCUSSION THREAD

**Discussion by Saddox** [probonopd/MiniDexed#521] [Q&A] [answered]
Hi!

I´ve just recieved a small PCM5102A module.
Connected it to my RP 2B and changed to I2S.
Like this:
https://imgur.com/a/QEyV4HV

SCK to ground, according to wiring. 

Red Led lights up on board. No smoke! That´s good. 
No sound.
Double check connections. Seems right.
Checking backside of PCM5102A module. H1L, H2L, H3L, H4L seems to be some settings (made with bridges according to Internet) but on my module there are no bridges. H3L seems to be some sort of softmute which should be HIGH to disable?

Any ideas?

**Comment by Gustavomurta** [comment]
Hi Saddox, 
sending some diagrams for verification.

https://github.com/Gustavomurta/miniDexed_RaspPi_synthesizer/tree/main/Hardware/PCM5102

![PCM5102 Raspberry Pi diagrama](https://github.com/probonopd/MiniDexed/assets/4587366/a9b9a5bf-13f3-4792-8bd0-64006c29bf97)

**Comment by Gustavomurta** [comment]
About jumpers settings:

![GY-PCM5102 diagrama](https://github.com/probonopd/MiniDexed/assets/4587366/22311cc3-ce9c-4dc4-8082-8e0713b745a6)


![GY-PCM5102 jumpers](https://github.com/probonopd/MiniDexed/assets/4587366/df24b4e4-509e-424e-b19b-20e16808b1f4)

**Comment by Gustavomurta** [comment]
See the jumpers on my PCM5102 module:

H3L must connected to 3.3V
Others must connected to GND. 

![GY-PCM5102 B](https://github.com/probonopd/MiniDexed/assets/4587366/e9a1210f-f272-4ed0-a508-b1bb3c9da7b4)

**Comment by Saddox** [comment]
Thanks! 

No jumpers connected at all on my PCM5102A-module. I will solder them tomorrow and see if I get some nice sound.

**Reply by Gustavomurta** [reply]
Is your module different from mine? Send a picture.

**Reply by Gustavomurta** [reply]
No jumpers, no sound.
Because the chip will not identify the format of the received data.
H4L => GND  (I2S format)

**Comment by Saddox** [comment]
Here are pictures of my module.

![baksida](https://github.com/probonopd/MiniDexed/assets/140212137/3337d188-ddf1-4d86-9215-e5dcdb53a963)
![Framsida](https://github.com/probonopd/MiniDexed/assets/140212137/bc15fb75-daa5-40cc-9e6e-32f0b928cb72)

**Reply by Banana71** [reply]
as you can see, you see nothing.  The corresponding bridges are probably missing.

**Reply by Saddox** [reply]
Yepp.. I will solder them when I get a new solder iron.. :)

**Comment by Saddox** [comment]
And also... my solder iron died.. after 10 years of service... :)

**Comment by Gustavomurta** [comment]
See I rotated the picture.
From my view of the board, these must be the jumpers.
To confirm, make sure the point on the left is GND. 
This way the point on the right will be the 3.3V. And the middle point is the pin to be bridged.

H3L must connected to 3.3V
Others must connected to GND.



![PCM5102 jumpers model](https://github.com/probonopd/MiniDexed/assets/4587366/f3410e15-8e97-48f6-88e7-d4af1c5faa8b)

**Comment by Gustavomurta** [comment, marked as answer]
To compare.
Jumpers are the same.
Labels are upside down.


![PCM5102 compararção](https://github.com/probonopd/MiniDexed/assets/4587366/31fd56bb-155e-4c7f-b0e9-2179ab54dc9f)

**Comment by Saddox** [comment]
Soldered the bridges. Works well!

Thank you!
